workspace/workflows/jobs.py

```diff
@@ -402,6 +402,19 @@ def get_text_blocks_for_key(args) -> str:
     return json.dumps(blocks)
 
 
+def get_usage_text(args) -> str:
+    return "\n".join(
+        [
+            "Below lists the behaviour for `show [target]`. The behaviour for `show-failed [target]` is the same, but skips repos whose workflows are all successful.",
+            "`show [all]`: Summarise all repos, sectioned by team.",
+            "`show [org]`: Summarise all repos for a known organisation, which is limited to the following shorthands and their full names: `os (opensafely)`, `osc (opensafely-core)` or `ebm (ebmdatalab)`.",
+            "`show [repo]`: Report status for all workflows in a known repo (e.g. `show airlock`) or a repo in a known org (e.g. `show os/some-study-repo`).",
```

Comment:
Since this is itself a job now, can we use the config list the repo options too? A normal slack user won't know what we mean by a "known" repo